index.html

```diff
@@ -17,16 +17,21 @@
     <header>
         <nav class="container-nav">
             <a href="#" class="logo-name">Federica Ulzurrun</a>
-            <div class="navlog">
-                <img src="img/navhamburger.png" alt="20px">
-            </div>
-            <ul class="nav-desk">
-                <li class="page-parts">portfolio</li>
-                <li class="page-parts">about</li>
-                <li class="page-parts">contact</li>
+            <ul class="nav-menu">
+                <li class="page-parts"><a class="nav-link" href="#portfolio">portfolio</a></li>
+                <li class="page-parts"><a href="#about" class="nav-link">about</a></li>
+                <li class="page-parts"><a href="#cont" class="nav-link">contact</a></li>
```

Comment:
- [ ] This is no doubt a nicely done job :hugs: but we just have to see how we can improve the styling to make it more similar to the Figma wireframe :hammer_and_wrench: 
- Right now, the blur effect used on the background as seen in the screenshot below is missing. I will suggest that you use a CSS style called `backdrop-filter: blur(xpx)`. You can read more about it on [W3Schools here](https://www.w3schools.com/cssref/css3_pr_backdrop-filter.php) 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/223244094-dbcfeea5-3f06-480c-960d-716b9059618e.png)|![image](https://user-images.githubusercontent.com/99715640/223244183-1afae061-fd42-451c-a167-1f6c08942c89.png)|